server/server/src/main/kotlin/org/jetbrains/bsp/bazel/server/bsp/managers/BazelBspAspectsManager.kt

```diff
@@ -74,7 +74,8 @@ class BazelBspAspectsManager(
       )
     val allowManualTargetsSyncFlags = if (shouldSyncManualFlags) listOf(buildManualTests()) else emptyList()
 
-    val flagsToUse = defaultFlags + allowManualTargetsSyncFlags
+    // --norun_validations because they make running this aspect 100x slower
+    val flagsToUse = defaultFlags + allowManualTargetsSyncFlags + listOf("--norun_validations")
```

Comment:
Should we add `--norun_validations` to `defaultFlags`?

Reply:
We may also want to add an explanation for why this flag can make the aspect run slower. Something along the lines of:

> Explicitly specifying a list of output groups should prevent Bazel from building the default output group and therefore should prevent it from running the actions generated by the targets provided:
https://bazel.build/reference/command-line-reference#flag--output_groups
>
> However, Bazel doesn't exclude the validations output group unless explicitly asked to do so, which could depend on the results of actions generated by the targets provided:
https://bazel.build/extending/rules#validation_actions

Reply:
Cool, I honestly didn't super understand what this was for, so that explanation is very helpful. :sweat_smile:

Reply:
Let me know if anything is still unclear :slightly_smiling_face: